repos/system_upgrade/el7toel8/actors/dnfshellrpmupgrade/actor.py

```diff
@@ -44,55 +27,11 @@ def process(self):
         # Make sure Subscription Manager OS Release is unset
         # cmd = ['subscription-manager', 'release', '--unset']
         # run(cmd)
-
-        shutil.copyfile(
-            self.get_file_path('rhel_upgrade.py'), '/lib/python2.7/site-packages/dnf-plugins/rhel_upgrade.py')
-
-        dnf_command = ['/usr/bin/dnf', 'rhel-upgrade', 'upgrade']
-
-        target_repoids = []
-        for target_repos in self.consume(UsedTargetRepositories):
-            for repo in target_repos.repos:
-                target_repoids.append(repo.repoid)
-
-        debugsolver = True if os.environ.get('LEAPP_DEBUG', '0') == '1' else False
-
-        shutil.copyfile(
-            '/etc/yum.repos.d/redhat.repo.upgrade',
-            '/etc/yum.repos.d/redhat.repo'
-        )
-
-        # FIXME: that's ugly hack, we should get info which file remove and
-        # + do it more nicely..
-        cmd = ['rm', '-f', '/etc/pki/product/69.pem']
```

Comment:
@pirat89  Feels strange, this shouldn't be necessary - /etc/pki/product-default/69.pem is installed by the release package and should get removed during the transaction.
So this seems off, that we have to add this for /etc/pki/product/69.pem. What's the reason for this being in product instead of product-default?

If we know that then we can check for this maybe. Anyway I have left this out for this moment until I get an answer.

Reply:
in short: `/etc/pki/product-default` is directory where we store the certificate of the system (RHEL product). IOW, the directory is sort of specific to RHEL systems. OTOH, `/etc/pki/product` is stadard path used by many other products (other RHEL products, 3rd-party products, ...). In case of RHEL, when you register a system, the certificate is copied to `/etc/pki/product` as there should be stored (let's say) "active" certificates. RHSM can store there additional certificates - based on SKUs you use. In case of update, the certificate in `/etc/pki/product-default` is updated, but not the one in `/etc/pki/product/` and the one stored here has higher priority. So you would not find the RHEL 8 system. It is possible that behaviour can be different, but this is one how it should behave and we've got recommendation to remove this one as well and ensure the new one is copied to the directory - that's something what we do not do yet if I am right, and we should.

Reply:
@pirat89 Ok thanks - Restored